ui/app/pages/send/send-footer/tests/send-footer-component.test.js

```diff
@@ -191,7 +191,7 @@ describe('SendFooter Component', function () {
     })
 
     it('should call history.push', function (done) {
-      Promise.resolve(wrapper.instance().onSubmit(MOCK_EVENT))
+      wrapper.instance().onSubmit(MOCK_EVENT)
```

Comment:
Can we make this whole test use `async`/`await`?

Reply:
Done.